.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "modules/core"]
+	path = modules/core
+	url = https://bitbucket.org/notabots/core.git
```

Comment:
I was considering the mirror but I had no rights to create mirror repo here + it would be extra work which is practically not as necessary because:

1) You can do it anytime in the future from any local checkout in case of any issue.
2) Service failure of bitbucket is as same probable as service failure of github.
3) I can give anyone all rights you need on bitbucket.
4) Maintenance of mirrors adds extra cost to any update.

So its valid idea, but extra work with no practical benefit now.

-----

missing licenses were added in [43c5d5f]

Reply:
OK